src/components/AddNote/AddNote.css

```diff
@@ -3,10 +3,63 @@
   border: none;
   resize: none;
 }
+.add-note {
+  position: relative;
+  color: black;
```

Comment:
Instead of using hardcoded value , you can create a color variable for black color too